test/integration/client-side-operations-timeout/node_csot.test.ts

```diff
@@ -808,6 +809,183 @@ describe('CSOT driver tests', metadata, () => {
     });
   });
 
+  describe('Change Streams', function () {
+    const metadata: MongoDBMetadataUI = { requires: { mongodb: '>=4.4', topology: '!single' } };
+    let internalClient: MongoClient;
+    let client: MongoClient;
+    let commandsStarted: CommandStartedEvent[];
+
+    beforeEach(async function () {
+      internalClient = this.configuration.newClient();
+      await internalClient
+        .db('db')
+        .dropCollection('coll')
+        .catch(() => null);
+      await internalClient.db('db').collection('coll').insertOne({ x: 0 });
+      commandsStarted = [];
+
+      client = await this.configuration.newClient(undefined, { monitorCommands: true }).connect();
+      client.on('commandStarted', ev => {
+        commandsStarted.push(ev);
+      });
+    });
+
+    afterEach(async function () {
+      await internalClient
+        .db()
+        .admin()
+        ?.command({ configureFailPoint: 'failCommand', mode: 'off' });
+      await internalClient?.close();
+      await client?.close();
+    });
+
+    context('when in stream mode', function () {
+      let data: any[];
+      let cs: ChangeStream;
+      let errorIter: AsyncIterableIterator<any[]>;
+
+      afterEach(async function () {
+        await cs?.close();
+      });
+
+      context('when the initial aggregate times out', function () {
+        beforeEach(async function () {
+          data = [];
+          const failpoint: FailPoint = {
+            configureFailPoint: 'failCommand',
+            mode: { times: 1 }, // fail twice to account for executeOperation's retry attempt
+            data: {
+              failCommands: ['aggregate'],
+              blockConnection: true,
+              blockTimeMS: 130
+            }
+          };
+
+          await internalClient.db().admin().command(failpoint);
```

Comment:
Just sharing - I added `configureFailPoint`/`clearFailPoint` helpers to `test/utils.ts` to make it easier to setup and teardown failpoints.